internal/pkg/manifest/env_test.go

```diff
@@ -522,6 +524,13 @@ http:
 			} else {
 				require.NoError(t, gotErr)
 				require.Equal(t, tc.wantedStruct, got)
+
+				roundtrip, err := yaml.Marshal(tc.wantedStruct)
+				require.NoError(t, err)
+				fmt.Printf("re-marshalled form:\n%s\n", string(roundtrip))
```

Comment:
We could in the future have unit test suites for `MarshalYAML` and compare the marshaled string there. For now I think this probably suffices for us to just visualize it.

Reply:
We can test that as well eventually - this catches most things by just checking for data loss in the round-trip.